packages/ebayui-core-react/src/ebay-file-preview-card-group/file-preview-card-group.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, FC, ComponentProps } from 'react'
+import { filterByType } from '../common/component-utils'
+import {
+    EbayFilePreviewCard,
+    EbayFilePreviewCardProps
+} from '../ebay-file-preview-card'
+
+export type EbayFilePreviewCardGroupProps = ComponentProps<'div'> & {
+    a11ySeeMoreText?: EbayFilePreviewCardProps['a11ySeeMoreText']
+}
+
+const SHOW_AMOUNT = 15 // default number of cards to show taken from marko's implementation
+
+const EbayFilePreviewGroup: FC<EbayFilePreviewCardGroupProps> = ({
+    a11ySeeMoreText,
+    children,
+    ...rest
+}) => {
+    const [cardsShowing, setCardsShowing] = useState(SHOW_AMOUNT)
+
+    const seeMore = () => {
+        setCardsShowing((prev) => prev + SHOW_AMOUNT)
+    }
+
+    const filePreviewCards = filterByType(children, EbayFilePreviewCard)
+    const notShowing = filePreviewCards.length - cardsShowing
+    const fileCardsToShow = filePreviewCards.slice(
+        0,
+        Math.min(cardsShowing, filePreviewCards.length)
+    )
+
+    return (
+        <div className="file-preview-card-group" {...rest}>
+            <ul>
+                {fileCardsToShow.map((previewCard) =>
+                    React.cloneElement(previewCard, {
+                        as: previewCard.props.as || 'li' // default in preview card is 'div', here should be 'li'
```

Comment:
@agliga in the marko version allows overwrite of the `as` property [here](https://github.com/eBay/evo-web/blob/main/packages/ebayui-core/src/components/ebay-file-preview-card-group/index.marko#L20). We are doing the same here, but is this by design or should it always be `li`?